sdk/build.gradle

```diff
@@ -81,73 +93,15 @@ artifacts {
     archives androidSourcesJar, androidJavadocsJar
 }
 
-publishing {
-    publications {
-        sdkPublication(MavenPublication) {
-            groupId "io.radar"
-            artifactId "sdk"
-            version radarVersion
-            artifact "$buildDir/outputs/aar/${project.getName()}-release.aar"
-            artifact androidSourcesJar
-            artifact androidJavadocsJar
-
-            pom {
-                name = "Radar Android SDK"
-                description = "Android SDK for Radar, the leading geofencing and location tracking platform"
-                url = "https://radar.io"
-
-                licenses {
-                    license {
-                        name = "Apache License, Version 2.0"
-                        url = "http://www.apache.org/licenses/LICENSE-2.0.txt"
-                    }
-                }
-
-                organization {
-                    name = "Radar Labs, Inc."
-                    url = "https://radar.io"
-                }
-
-                developers {
-                    developer {
-                        id = "radarlabs"
-                        name = "Radar Labs, Inc."
-                    }
-                }
-
-                scm {
-                    connection = "scm:git:git://github.com:radarlabs/radar-sdk-android.git"
-                    developerConnection = "scm:git:ssh://github.com:radarlabs/radar-sdk-android.git"
-                    url = "https://github.com/radarlabs/radar-sdk-android"
-                }
-
-                withXml {
-                    def dependenciesNode = asNode().appendNode('dependencies')
-                    configurations.implementation.allDependencies.each { dependency ->
-                        if (dependency.group == null || dependency.name == null || dependency.version == null) {
-                            return
-                        }
-                        def dependencyNode = dependenciesNode.appendNode('dependency')
-                        dependencyNode.appendNode('groupId', dependency.group)
-                        dependencyNode.appendNode('artifactId', dependency.name)
-                        dependencyNode.appendNode('version', dependency.version)
-                    }
-                }
-            }
-        }
+mvnpublish {
+    publication {
+        name = 'Radar Android SDK'
+        description = 'Android SDK for Radar, the leading geofencing and location tracking platform'
+        repositoryName = 'radar-sdk-android'
+        group = 'io.radar'
+        artifactId = 'sdk'
+        version = "$radarVersion$buildNumber"
+        artifacts = ["$buildDir/outputs/aar/$project.name-release.aar", androidSourcesJar, androidJavadocsJar]
+        server = MavenServer.STAGING
```

Comment:
Should this be `MavenServer.RELEASE`?

Reply:
Updated. Also kept this server configuration, so that we can push to staging without releasing if we ever want to.